Library/Homebrew/utils/github.rb

```diff
@@ -530,7 +534,7 @@ def create_bump_pr(info, args:)
                     "--", *changed_files
         return if args.commit?
 
-        safe_system "git", "push", "--set-upstream", remote_url, "#{branch}:#{branch}"
+        system_command("git", args: ["push", "--set-upstream", remote_url, "#{branch}:#{branch}"], print_stdout: true)
```

Comment:
```suggestion
        system_command!("git", args: ["push", "--set-upstream", remote_url, "#{branch}:#{branch}"], print_stdout: true)
```

Needed I think to emulate `safe_system` otherwise it won't `raise` on failure, right?

Reply:
Ah, right. Yes. Forgot that I need exclamation mark or else it will just save results etc. for later.